metaseq/launcher/tombyard.py

```diff
@@ -111,3 +116,25 @@ def tombstones(job_id, base_dir, period=datetime.timedelta(seconds=60), dirstone
         target=tombstones_procedure, args=(job_id, dirstones, period), daemon=False
     )
     heartbeat_proc.start()
+
+
+if __name__ == "__main__":
+    logger = logging.getLogger(f"touchstoning process: PID {os.getpid()}")
+    logger.setLevel(logging.DEBUG)
+    parser = argparse.ArgumentParser("Tombstoning script")
+    parser.add_argument(
+        "-bd",
+        "--base-dir",
+        type=str,
+        required=True,
+        help="number of random hyperparam configurations to try (-1 for grid search)",
+    )
+    parser.add_argument(
+        "-j",
+        "--job-id",
+        type=int,
+        required=True,
+        help="number of random hyperparam configurations to try (-1 for grid search)",
+    )
+    args = parser.parse_args()
+    tombstones(str(args.job_id), args.base_dir, logger=logger)
```

Comment:
Seeing this error show up:
```
Traceback (most recent call last):
  File "/home/susanz/.conda/envs/metaseq-20221105/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/susanz/.conda/envs/metaseq-20221105/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/checkpoint/z/checkpoints/susanz/slurm_snapshot_code_oss/2022-11-11T13_10_36.631064/metaseq/launcher/tombyard.py", line 140, in <module>
    tombstones(str(args.job_id), args.base_dir, logger=logger)
TypeError: tombstones() got an unexpected keyword argument 'logger'
```

Reply:
Uff, sorry, forgot to change this line. Now it matches Igor's file https://github.com/fairinternal/metaseq-internal/blob/main/metaseq_internal/fb_sweep/tombyard.py exactly, except for formatting.